layers/+source-control/github/README.org

```diff
@@ -31,16 +31,22 @@ add =github= to the existing =dotspacemacs-configuration-layers= list in this
 file.
 
 ** Git configuration
-You will need to generate a [[https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line][personal access token]] on GitHub. This token should
-have the =gist= and =repo= permissions. Once this token is created, it needs to
-be added to your =~/.gitconfig=
+The GitHub layer (forge & github-clone) uses a [[https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line][personal access token]] to access
+GitHub. The token should have the =repo= permission.
 
-You will also need to [[https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/][generate an SSH key]] and [[https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/][add it to your GitHub account]].
+Add the token and GitHub account username to your =~/.gitconfig=
 
 #+BEGIN_SRC sh
+  git config --global github.user <github-username>
   git config --global github.oauth-token <token>
 #+END_SRC
 
+[[https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/][Generate an SSH key]] and [[https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/][add it to your GitHub account]] to remove the need
+to provide GitHub login details for each request from Magit to GitHub.
```

Comment:
s/Magit/Magit Forge ?

Reply:
Yes, this needs a little more elaboration.  An SSH key should be used when using SSH URLs for remote repositories with Magit.  The SSH key is not needed for Forge.  
I've pushed an update to this section to clarify.